### PR TITLE
Changes done 

### PR DESCRIPTION
**Pull request by reiwamktr**
sdfsdfsdf

### CODE REVIEW

**Review by neerajan007 — COMMENTED** [review]
Taile garekai bhaxaina

File: cv/js-files/hi.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+yo maile k gariraxu hai
```
Comment (neerajan007) [inline cv/js-files/hi.js:1]: kasle lekhxa yesto tori jasto

File: cv/js-files/hi.js [hunk 2]
```diff
@@ -0,0 +1,3 @@
+yo maile k gariraxu hai 
+
```
Comment (neerajan007) [inline cv/js-files/hi.js:2]: yo nmee bha xaina hai

File: cv/js-files/hi.js (hunk 1 above)
Comment (neerajan007) [inline cv/js-files/hi.js:1]: yo nee khasai bhaxaina hai

**Review by neerajan007 — COMMENTED** [review]
la her taw changes haru

File: cv/js-files/hi.js (hunk 2 above)
Comment (reiwamktr) [reply cv/js-files/hi.js:2]: hain yo taw hunu parne ho taw

File: cv/js-files/hi.js (hunk 1 above)
Comment (reiwamktr) [reply cv/js-files/hi.js:1]: bhaisakyo

**Review by neerajan007 — APPROVED** [review]
la la approve garidinxu

**Comment by reiwamktr** [comment]
Thanks

**Pull request merged**